### ISSUE TITLE
get ingredients with allergen

### ISSUE THREAD

**Issue by DanteIka** [openfoodfacts/openfoodfacts-php#30]
I tried to get ingredients with allergen by search string
For example when I type the "bread", all breads should be come out and have allergen tag
How can I implement this?

**Comment by Dwarfex** [comment]
You should be able to achive this with the

getByFacets(array $query = [], int $page = 1) function of the API 

$firstPage    = $api->getByFacets(['allergens'=> 'bread']);    

(although i believe there is no allergen named "Bread" - you could try "gluten")
Let me know if this works for you  - i'll add an example to the readme.

**Comment by DanteIka** [comment]
Thanks for your email.

I tried like that but the error return

Message: Client error: `GET
https://fr.openfoodfacts.org/allergens/bread/1.json` resulted in a `404 Not
Found` response:

This is the error result

Please help me how can I get

**Comment by Dwarfex** [comment]
yeah i've looked into it - it seems as if the api has changed and the package here needs to be updated - i'm on it

**Comment by DanteIka** [comment]
Thanks for your helping
